libpod/runtime_img.go

```diff
@@ -33,9 +33,9 @@ import (
 // Runtime API
 
 const (
-	// DefaultRegistry is a prefix that we apply to an image name
+	// DefaultTransport is a prefix that we apply to an image name
 	// to check docker hub first for the image
-	DefaultRegistry = "docker://"
+	DefaultTransport = "docker://"
```

Comment:
As long as you are at it, line 54ish in runtime.go has a struct that would benefit from this change too.